src/xcode/ENA/ENA/Source/Client/__tests__/Mocks/CachingHTTPClientMock.swift

```diff
@@ -48,6 +48,18 @@ final class CachingHTTPClientMock: CachingHTTPClient {
 		}
 		return configMetadata
 	}()
+	
+	static let staticLocalStatistics: SAP_Internal_Stats_LocalStatistics = {
+		guard
+			let url = Bundle(for: CachingHTTPClientMock.self).url(forResource: "default_local_statistics", withExtension: "bin"),
+			let data = try? Data(contentsOf: url),
+			let localStatistics = try? SAP_Internal_Stats_LocalStatistics(serializedData: data)
+		else {
+			fatalError("Cannot initialize static test data")
```

Comment:
Please don't use `fatalError` in tests! This might/will kill the test runner and lead to all kinds of follow up issues. Create a function that throws an error in case of resources not found.